notebooks/icrp107_dataset/icrp107_dataset.ipynb

```diff
@@ -284,23 +379,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Download ICRP-107 Supplemental Material and read data from ICRP-07.NDX file line by line into the DataFrame."
+    "Read data from ICRP-107 file (`ICRP-07.NDX`) line by line into the DataFrame. For atomic masses of metastable states, pull isomer excitation energy from NuBase dataframe."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 7,
    "metadata": {
     "scrolled": false
    },
    "outputs": [],
    "source": [
-    "url = \"https://journals.sagepub.com/doi/suppl/10.1177/ANIB_38_3/suppl_file/P107JAICRP_38_3_Nuclear_Decay_Data_suppl_data.zip\"\n",
-    "r = requests.get(url)\n",
-    "z = zipfile.ZipFile(io.BytesIO(r.content))\n",
-    "z.extractall()\n",
-    "\n",
-    "file_NDX = open(\"P 107 JAICRP 38(3) Nuclear Decay Data for Dosimetric Calculations(supplementary data)/ICRP-07.NDX\", \"r\")\n",
+    "#file_NDX = open(\"P 107 JAICRP 38(3) Nuclear Decay Data for Dosimetric Calculations(supplementary data)/ICRP-07.NDX\", \"r\",encoding=\"ISO-8859-1\")\n",
```

Comment:
This line can be deleted due to switching to holding ICRP-07.NDX within the repository. This chance also means you can delete the imports for io, shutil, zipfile and requests in the first code cell, as these libraries are no longer used.